packages/config/config/devices/0x000c/hs-wx300.json

```
@@ -161,6 +161,81 @@
 					"value": 1
 				}
 			]
+		},
+		{
+			"#": "33",
+			"label": "Startup Flash Behavior",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "LEDs do not flash on startup",
+					"value": 0
+				},
+				{
+					"label": "LEDs flash on startup to indicate switch or dimmer operation",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "34",
+			"label": "Relative LED Indicator Brightness",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 6,
+			"defaultValue": 3,
+			"unsigned": true
+		},
+		{
+			"#": "35",
+			"label": "Paddle Behavior for Switch Mode (Toggle Feature)",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Top paddle press turns load ON. Bottom paddle press turns load OFF.",
+					"value": 0
+				},
+				{
+					"label": "Either paddle press turns load ON if OFF and OFF if ON.",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "36",
+			"label": "Default Dim Value when turned ON via paddle press",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 99,
+			"defaultValue": 0,
+			"unsigned": true
```

```suggestion
			"options": [
				{
					"label": "Last non-zero brightness",
					"value": 0
				}
			]
```
unsigned isn't necessary unless the check complains :)